coresdk/src/external/civetweb.h

```diff
@@ -0,0 +1,1035 @@
+/* Copyright (c) 2013-2016 the Civetweb developers
```

Comment:
It would be good to have a way of importing these (and sqlite) without having the code in our repo... much like SDL... are these available on a repo we could pull them from?